bootstrap/pesticide_seed.py

```
@@ -0,0 +1,1179 @@
+    if os.environ.get("FARMOS_PESTICIDE_SEED_REEXEC") == "1":
+        return
+
+    backend_python = Path(_venv_python(BACKEND_DIR)).resolve()
+    current_python = Path(sys.executable).resolve()
+    if current_python == backend_python:
+        return
+
+    env = os.environ.copy()
+    env["FARMOS_PESTICIDE_SEED_REEXEC"] = "1"
+    cmd = [str(backend_python), str(Path(__file__).resolve()), *sys.argv[1:]]
+    result = subprocess.run(cmd, cwd=str(PROJECT_ROOT), env=env)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)